components/pedigree/resources/src/main/resources/pedigree/nodeMenuFields.js

```diff
@@ -311,7 +311,7 @@ define([
             },
             {
                 'name' : 'setproband',
-                'label' : 'Set this person to be the proband node',
+                'label' : 'This person is the proband',
```

Comment:
When there is a button with this text I think it can be confused with a statement (e.g. it may look like an _affirmation_ "yes, this patient is the proband" as opposed to _action_ "make this patient the proband")

Reply:
Right. Which reminds me I wanted to change this button to a checkbox. For now I'll change the label ti "make this...". Thanks.